woocommerce/woocommerce
# Add/edit products: add new Tracks props

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds a few props to track the events `wcadmin_product_add_publish` and `wcadmin_product_update`.

Closes #32674.

### How to test the changes in this Pull Request:

1. Enable debug messages inside browser devtools (P90Yrv-1Wj-p2#debug-devtools)
2. Verify in your browser console that the following prop list is being sent correctly for the event `wcadmin_product_update`.
 
- description: Yes|No
- short_description: Yes|No
- product_image: Yes|No
- product_gallery: Int (count of images in the gallery)
- categories: Int (count of categories)
- tags: Int (count of tags)
- sale_price: Yes|No
- weight: Yes|No
- dimensions: Yes|No
- upsells: Yes|No
- cross-sells: Yes|No
- attributes: Int (count of attributes)
- purchase_note: Yes|No
- menu_order: Yes|No
- enable_reviews: Yes|No

3. Install [WooCommerce Admin Test Helper](https://github.com/woocommerce/woocommerce-admin-test-helper).
4. Create a new product.
5. Go to WooCommerce -> Status -> Logs (URL wp-admin/admin.php?page=wc-status&tab=logs).
6. Select a tracks-* log from the dropdown.
7. Verify that the event `wcadmin_product_add_publish` is sending the props described in step 2 correctly.

### Other information:




### FOR PR REVIEWER ONLY:
